src/js/rx/containers/Active.jsx

```diff
@@ -1,17 +1,72 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { browserHistory } from 'react-router';
 
 import PrescriptionList from '../components/PrescriptionList';
+import PrintList from '../components/PrintList';
+import SortMenu from '../components/SortMenu';
+
 import { loadData } from '../actions/prescriptions.js';
 
 class Active extends React.Component {
   componentWillMount() {
     this.props.dispatch(loadData());
+    this.handleSortOnChange = this.handleSortOnChange.bind(this);
+    this.handleSortOnClick = this.handleSortOnClick.bind(this);
+    this.dispatchSortAction = this.dispatchSortAction.bind(this);
+  }
+
+  dispatchSortAction(action) {
+    this.props.dispatch({ type: action });
+  }
+
+  handleSortOnChange(domEvent) {
+    if (domEvent.type === 'change') {
+      browserHistory.push({
+        pathname: '/rx',
+        query: {
+          sort: domEvent.target.value
+        }
+      });
+    }
+    this.dispatchSortAction(domEvent.target.value);
+  }
+
+  handleSortOnClick(domEvent) {
+    const fullURL = domEvent.target.href;
+
+    // Find the sort parameter, split the query string on the = and retrieve value
+    const sortParam = fullURL.match(/sort=[-a-z]{1,}/i)[0].split('=')[1];
```

Comment:
My preference would be that `react-router` handles URL params, since it already has it built in.

That would mean a slightly different approach here-- Rather than intercepting the click event, we would have to wait until the actual `window.location` changed, then fire off an action.

[react-router-redux](https://github.com/reactjs/react-router-redux) seems to handle this exact situation.  It would be a little upfront work to set up, but I think it's the right solution.  One fewer code path, anyway.